issuing_authorization.go

```diff
@@ -235,7 +177,7 @@ type IssuingAuthorization struct {
 	Livemode            bool                                    `json:"livemode"`
 	MerchantAmount      int64                                   `json:"merchant_amount"`
 	MerchantCurrency    Currency                                `json:"merchant_currency"`
-	MerchantData        *IssuingMerchantData                    `json:"merchant_data"`
+	MerchantData        *IssuingAuthoriationMerchantData        `json:"merchant_data"`
```

Comment:
Typo in the name: `IssuingAuthoriationMerchantData` (missing a `z`)

Reply:
damn nice catch. Fixed PTAL